kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQuerySchemaConverter.java

```diff
@@ -178,6 +179,9 @@ private void throwOnCycle(Schema kafkaConnectSchema, List<Schema> seenSoFar) {
       setNullability(kafkaConnectSchema, res);
       if (kafkaConnectSchema.doc() != null) {
         res.setDescription(kafkaConnectSchema.doc());
+      } else if (kafkaConnectSchema.parameters() != null &&
+              kafkaConnectSchema.parameters().get(AVRO_DOC_PARAMETER + fieldName) != null){
+        res.setDescription(kafkaConnectSchema.parameters().get(AVRO_DOC_PARAMETER + fieldName));
```

Comment:
We're introducing Avro converter-specific logic into the connector here. That doesn't feel right; connectors should be largely agnostic about the converter that they're configured with, with the possible exception of requiring schemas with the data that they process.

If we add this logic here, what's to stop other converter developers from filing similar PRs against this and other connector repos to add more converter-specific logic?

Do you know why the Avro converter was changed in such a way that causes a property that used to be set as the doc of the Connect schema to be set instead as a converter-specific property, leaving the doc blank? The linked PR's description is a little thin and the accompanying issue is pretty lengthy. It'd be nice if you could provide a summary of the motivation for that change (if you're already aware of the rationale), or maybe bring in someone who worked on it to help explain things here.

As things stand right now, I'm hesitant to include this change and think that an adjustment of the converter logic may be warranted in its place.